src/platforms/flutter/usage/advanced-usage.mdx

```diff
@@ -65,3 +65,14 @@ Always prefer the `SentryFlutter.init(...)` instead of `Sentry.init(...)` as it
 ## Dart environment variables
 
 - You can configure the `SENTRY_DSN`, `SENTRY_RELEASE`, `SENTRY_DIST` and `SENTRY_ENVIRONMENT` via the Dart environment variables passing the `--dart-define` flag to the compiler.
+
+## Building specific ABI on Android
+
+The Sentry Flutter SDK includes Sentry Android SDK which bundles multiple native libraries, for multiple ABIs.
+When building a Flutter app for a specific ABI using the `--target-platfort` argument, for example an ARM 32bit apk:
+
+```sh
+flutter build apk --target-platform=android-arm --split-per-abi
+```
+
+Make sure to include the `--split-per-abi` flag.
```

Comment:
remove this line if using the edit above

Reply:
```suggestion
```